add OYDID driver

fabianekc opened the pull request:
this is the very first version of a driver - see initial documentation: https://github.com/OwnYourData/oydid/blob/main/uni-registrar-driver-did-oyd/README.md

peacekeeper commented on config.json:
```diff
@@ -27,6 +27,10 @@
 			"propertiesEndpoint": "true"
+		}, {
+			"method": "oyd",
+			"url": "https://oydid-registrar.data-container.net/",
```
Shouldn't this point to the local Docker container (e.g. `http://driver-did-oyd:3000/`) instead of a remote URL?

fabianekc replied:
OK - updated to local Docker instance
(Uniresolver uses the remote instance hosted by OwnYourData and therefore, I thought it will be the same for Uniregistrar)

peacekeeper commented:
As discussed on today's call, 2 small changes:
- "identifier" field should be renamed to "did"
- "jobId" is not necessary if "didState.state" == "finished"

Otherwise looks good I think!

fabianekc commented:
already updated driver and test cases - should work fine

fabianekc commented:
Thanks for identifying that empty DID documents did not work. This is now fixed and the provided example works with the latest Docker image (https://hub.docker.com/r/oydeu/oydid-registrar - `oydeu/oydid-registrar:220405` and `oydeu/oydid-registrar:latest`); also updated instance running at `https://oydid-registrar.data-container.net/`
